### DISCUSSION TITLE
Post REACH

### DISCUSSION THREAD

**Discussion by anurag6713** [mattermost/community#141] [Practical Ideas]
VERY VERY LATE TO THE PARTY 😅

Know how many users in a channel has read a particular post. This info can be only visible to the admin/author. Unlike WhatsApp Read receipt, REACH only deals with numbers without affecting privacy.

Example: Like `40/100` (40 users have read this message out of 100) or `40%`

**Comment by esethna** [comment]
AFAIK this is a major performance challenge, but we do have a read reciepts system planned for DMs/GMs with less users: https://mattermost.atlassian.net/browse/MM-36529

**Reply by esethna** [reply]
Not that I wouldn't like to see this! Just sharing the feedback I've heard about this kind of a feature

**Reply by Willyfrog** [reply]
if this is something that is not constantly happening but only showing if the user ask for it, we could probably check membership last read to get the number.

For constantly showing, then it would probably be a performance problem and most of the time wouldn't be used.

**Reply by jespino** [reply]
I like the pov of Guillermo, clearly a good option to just check that when needed
